azure-pipelines.yml

```diff
@@ -43,23 +46,27 @@ stages:
         projectKey: '2011-fakebook-project3_posts'
         extraProperties: |
           sonar.cs.opencover.reportsPaths=$(Agent.TempDirectory)/**/coverage.opencover.xml
-          sonar.cs.vstest.reportsPaths=$(Agent.TempDirectory)/*.trx
+
     - task: UseDotNet@2
       displayName: 'Use .NET 5 SDK'
       inputs:
-        packageType: 'sdk'
-        version: '5.x'
-        installationPath: $(Agent.ToolsDirectory)/dotnet
-        workingDirectory: 'Fakebook.Posts.RestApi/'
+        workingDirectory: '$(Agent.ToolsDirectory)/dotnet'
+
+    - script: dotnet build --configuration $(buildConfiguration)
+      displayName: dotnet build
+      workingDirectory: $(targetSolution)
```

Comment:
and yeah fix this like the error says

Reply:
$(directory) isn't right, the whole point of that step is to ensure we build the whole solution.
"directory" is a badvariable name for this reason, ambiguous

Reply:
removed directory variable